junit4-runner/src/main/java/org/junit/gen5/junit4/runner/JUnit5.java

```diff
@@ -131,38 +130,29 @@ private void addFiltersFromAnnotations(DiscoveryRequest spec) {
 	}
 
 	private List<DiscoverySelector> getPackageSpecificationElements() {
-		return stream(getPackageNames()).map(PackageTestPlanSpecificationElementBuilder::byPackageName).collect(
-			toList());
+		return stream(getPackageNames()).map(PackageSelectorBuilder::byPackageName).collect(toList());
```

Comment:
Not sure why JUnit5 should use the builders from dsl package. This creates an extra dependency - especially if the dsl should move to a seperate project - without simplifying the code.
`PackageSelector::new` would be even shorter than `PackageSelectorBuilder::byPackageName`

And - what's more important - the dsl is supposed to be less stable than the underlying abstractions.